index.html

```diff
@@ -0,0 +1,555 @@
+<!DOCTYPE html>
+<html lang="en"> 
+    <head>
+        <meta charset="utf-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <title>TNW Clone</title>
+        <meta name="description" content="">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/semantic-ui/2.4.1/semantic.min.css">
+        <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
+        <link rel="stylesheet" href="css/main.css">
+        <link rel="stylesheet" type="text/css" media="screen and (max-width: 1025px) and (min-width: 769px)" href="css/tablet.css">
+        <link rel="stylesheet" type="text/css" media="screen and (max-width: 768px)" href="css/mobile.css">  
+    </head>
+    <body>
+        <nav>
+            <div class="top-nav">
+                <ul>
+                    <li><a href="#">News</a></li>
+                    <li><a href="#">TNW2020</a></li>
+                    <li><a href="#">Business <i class="angle down icon"></i></a></li>
+                    <li><a href="#">AMAs</a></li>
+                    <li><a href="#">Spaces</a></li>
+                    <li><a href="#">Terms & Conditions</a></li>
+                </ul>
+                <ul>
+                    <li><a href="#">About</a></li>
+                    <li><a href="#">Advertise</a></li>
+                    <li><a href="#">Jobs</a></li>
+                    <li><a href="#">Contact</a></li>
+                </ul>
+            </div>  
+        </nav>
+        <main>
+            <div class="fixed-nav">
+                <div class="bottom-nav">
+                    <div class="tnw-logo">
+                        <img alt="TNW-logo" src="https://cdn0.tnwcdn.com/wp-content/themes/cyberdelia/assets/img/tnw.svg">
+                    </div>
```

Comment:
- [ ] The logo shouldn't appear in the large breakpoint of your page.
![Screenshot from 2020-03-18 11-53-28](https://user-images.githubusercontent.com/51926937/76991949-13c08a00-6910-11ea-8f9b-6f2e46d4afab.png)

Original:
![Screenshot from 2020-03-18 11-53-34](https://user-images.githubusercontent.com/51926937/76991963-18853e00-6910-11ea-8a5a-0b6077cffd0c.png)